assets/src/edit-story/panels/shared.js

```diff
@@ -79,6 +79,45 @@ InputGroup.propTypes = {
 InputGroup.defaultProps = {
 	type: 'number',
 	postfix: '',
+	disabled: false,
+};
+
+function SelectMenu( { label, options, value, onChange, postfix, disabled } ) {
+	return (
+		<Group disabled={ disabled }>
+			<Label>
+				{ label }
+			</Label>
+			<select
```

Comment:
Should we use styled-components here instead for consistency?

Reply:
This select is a temp element for now that soon will be replaced. I am not sure I see the point of styling these select nicely nice now.

Reply:
Agreed that no point stying nicely, was thinking rather using something just like this (since it's used elsewhere everywhere as well, even if there's no styling at all):

`const Select = styled.select``;` or 
```
const Select = styled.select`
width: 100%;
`;
```

Reply:
Changed in https://github.com/ampproject/amp-wp/pull/3998/commits/6f37350ad93201af7003de73c8a31cfb068684ef